### ISSUE TITLE
[Suggestion] Front Loader attachment for Steam Tractor

### ISSUE THREAD

**Issue by Blanclour** [StrangeLoopGames/EcoSuggestions#771]
Currently the early game has carts to help facilitate the moving of dirt and resources while the end game has powerful skid steers and excavators to level whole regions easily. The mid game possesses steam trucks capable of moving resources more efficiently but we are still stuck shoveling dirt by hand!

The mid game is a time of industrialization where certain tasks are becoming easier, especially the transport of resources. Towns are starting to grow with the increasing availability of building materials, but people are still stuck shoveling dirt by hand to make a footprint for each building and pathways for roads.

We need a way to move dirt more easily in the mid game, I suggest a simple attachment for the steam tractor that makes it function similar to a skid steer. The balance of this would be greatly reduced bucket capacity compared to the skid steer and excavator, and the inability to break anything hard such as rock, or ores. With such an attachment, the steam tractor would make the life of mid game road builders and town planners much easier.

**Comment by johnkslg** [comment]
@craig-jong @keegan-o good idea for thinking about future vehicles, would be easy to do as an attachment.
